modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -272,24 +274,41 @@ definition {
 	}
 
 	@description = "LPS-178192. Verifies only if the setting has never been changed by the user, either when signed in or not signed in, will the default value be displayed.."
-	@ignore = "Test Stub"
 	@priority = 5
 	test UntoggledSettingsAreAppliedImmediately {
 		property portal.acceptance = "true";
 
-		//task ("Given When Then") {
+		task ("And toggle show underline to ON state") {
+			AccessibilityMenu.openAccessibilityMenu();
 
-		// // Given home page
-		// // And underline class is applied to body element
-		// // // verify show underline config in accessibility menu is on
-		// // // verify body element does have "c-prefers-link-underline" class present
-		// // When access Accessibility Menu
-		// // And toggle OFF show underline effect in links
-		// // Then Powered by Liferay link does not have underline class applied
-		// // // assert body element does not have "c-prefers-link-underline" class present
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
 
-		//}
+		task ("And underline class is applied to body element") {
+			AssertChecked.assertCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
 
+			AssertHTMLSourceTextPresent(value1 = "c-prefers-link-underline");
```

Comment:
Please target the body class specifically with `AssertElementPresent(`
locator is something like: `//body(contains...c-prefers-link-underline...)`

We want to make sure the class is applied globally on the page so we want to verify that the class is at the body element level. Not just anywhere in the html source.

Reply:
I tried `AssertHTMLSourceTextPresent(value1 = "//body[contains(@class, 'c-prefers-link-underline')]");` 
But it fails... 😢

Reply:
Also tried other options like `	AssertHTMLSourceTextPresent.assertHTMLSourceTextPresentNoSPARefresh(value1 = "c-prefers-link-underline");` and got the same fail

Reply:
What about just `AssertElementPresent`?

Reply:
I thought `AssertElementPresent` was not used for HTML actually...

Reply:
Aright, done!